src/frontend/pages/home/home.js

```diff
@@ -1,4 +1,3 @@
-import { useNavigate } from "react-router-dom";
 import { Footer, Navbar} from "../../components/index";
```

Comment:
If you are doing barrel exports, you really don't need to specify `index` at the end. Just the folder name is enough.
Browser/builder will automatically look into index.js if it is present in that folder.

Reply:
oh i wasn't aware about this. thank, now onward i ll do the same

Reply:
refactored barrel import